api/src/Vendors/Stripe/PostPayment.php

```diff
@@ -0,0 +1,48 @@
+<?php declare(strict_types=1);
+
+namespace Vendor\Payment;
+
+use Slim\Http\Request;
+use Slim\Http\Response;
+use App\Controller\NotFoundException;
+
+class PostPayment extends BasePayment
+{
+
+
+    public function buildResource(Request $request, Response $response, $params): array
+    {
+        $required = ['success', 'cancel', 'price', 'quantity', 'item'];
+        $body = $this->checkRequiredBody($request, $required);
+
+        $checkout_session = \Stripe\Checkout\Session::create([
+            'payment_method_types' => ['card'],
+            'line_items' => [[
```

Comment:
So, right now, this appears to be fixed at a single line item; we will want to iterate over the line items in a cart.

Reply:
This also means that this endpoint could actually just take a shopping_cart_id rather than take the details as part of the POST, if I understand the workflow.